### DISCUSSION TITLE
Update Slayer Target Highlighting

### DISCUSSION THREAD

**Discussion by MoreBuchus** [runelite/runelite#15185] [Ideas]
With the recent changes to NPC Indicators, I feel like Slayer should get some love too. Currently, you can only highlight NPCs with tile, hull, outline, and select the color. Unlike NPC Indicators, there is none of the other highlighting styles, as well as options for highlight width, outline feather, a fill color, and the option to ignore dead targets (currently the highlight stays until npc despawned).

Current slayer plugin:
![current slayer](https://i.imgur.com/Uc57YOy.gif)

Proposed slayer changes:
![proposed slayer](https://i.imgur.com/kVzqsCK.gif)

**Comment by ShaunDreclin** [comment]
Came here to suggest this, I particularly want true tile + southwest true tile options for slayer highlights

**Comment by Callum-Bartle-92019337** [comment]
South west tile would be amazing in the slayer plugin
I'm always having to tag my slayer task then untag when done for this

**Comment by IsekaiVictim** [comment]
So many slayer monsters have janky animation stalls that would be greatly improved by the addition of true tile and sw true tile
